pkg/frr/dbus/dbus.go

```diff
@@ -0,0 +1,29 @@
+package dbus
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/coreos/go-systemd/v22/dbus"
+)
+
+//go:generate mockgen -destination ./mock/mock_frr.go . SystemInterface,ConnInterface
+type SystemInterface interface {
+	NewConn(ctx context.Context) (ConnInterface, error)
+}
+
+type ConnInterface interface {
```

Comment:
If there is nothing else using the name `Connection`, I'd just call it that instead of adding the `Interface` suffix. Same for `System`.

Reply:
Done